araproc/analysis/standard_reco.py

```diff
@@ -1287,22 +1295,24 @@ def get_csw(
             roll_shift_time = roll_shift_bins*(times[1] - times[0])
             if abs(roll_shift_bins) % 1.0 > 0.0001: 
                 # roll_shift is not close to an integer. Add to the warning
-                warning += 10
+                warning += 1_00
             roll_shift_bins = int(roll_shift_bins)
             if abs(roll_shift_bins)>len(times) and not is_software:
                 # More waveform to roll than there is time in the waveform,
                 #   so add to the warning tracker. 
                 # Software triggers are so short, this sometimes occurs for them.
                 #   Don't warn in this scenario.
-                warning += 10_00_00_00
+                warning += 1_00_00
             if roll_shift_bins > 0 and abs(roll_shift_bins)<len(times): 
-                # Rolling from front to back, check that signal region isn't in the front
+                # Rolling from front to back, warn if the expected signal is
+                #   in the region about to be rolled
                 if times[0] <= expected_signal_time <= times[roll_shift_bins]: 
-                    warning += 10_00
+                    warning += 1_00_00_00
             elif roll_shift_bins < 0 and abs(roll_shift_bins)<len(times): 
-                # Rolling from back to front, check that signal region isn't in the back
+                # Rolling from back to front, warn if the expected signal is
+                #   in the region about to be rolled
                 if  times[roll_shift_bins]  <= expected_signal_time <= times[-1]: 
-                    warning += 10_00
+                    warning += 1_00_00_00
```

Comment:
is this supposed to be different from the one above? (idk if I'm following the logic of these warning, but the rest are different from each other)

Reply:
I figured these are similar enough (since they both involve rolling the region with expected signal) but they can have different numbers too

Reply:
Fine with me. Just wanted to make sure it was intended. I'm happy either way.